src/theory/sets/theory_sets_rels.cpp

```diff
@@ -32,24 +32,22 @@ typedef std::map< Node, std::unordered_set< Node, NodeHashFunction > >::iterator
 typedef std::map< Node, std::map< kind::Kind_t, std::vector< Node > > >::iterator               TERM_IT;
 typedef std::map< Node, std::map< Node, std::unordered_set< Node, NodeHashFunction > > >::iterator   TC_IT;
 
-TheorySetsRels::TheorySetsRels(context::Context* c,
-                               context::UserContext* u,
-                               eq::EqualityEngine* eq,
-                               TheorySetsPrivate& set)
-    : d_eqEngine(eq),
-      d_sets_theory(set),
-      d_trueNode(NodeManager::currentNM()->mkConst<bool>(true)),
-      d_falseNode(NodeManager::currentNM()->mkConst<bool>(false)),
-      d_shared_terms(u),
-      d_satContext(c)
+TheorySetsRels::TheorySetsRels(SolverState& s,
+                               InferenceManager& im,
+                               eq::EqualityEngine& e,
+                               context::Context* c,
```

Comment:
CLion detected that this argument `c` is never used,  and it can be removed from the constructor. Previously it was used to initialize the deleted field `d_satContext` which was never used.